sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsClient.java

```diff
@@ -56,74 +54,76 @@
     }
 
     /**
-     * Returns the detected language and a numeric score between zero and one. Scores close to one indicate 100%
+     * Returns the detected language and a confidence score between zero and one. Scores close to one indicate 100%
      * certainty that the identified language is true.
      *
      * <p><strong>Code Sample</strong></p>
-     * <p>Detects the language of single input text</p>
+     * <p>Detects the language of single input text.</p>
      * {@codesnippet com.azure.ai.textanalytics.TextAnalyticsClient.detectLanguage#String}
      *
      * @param text The text to be analyzed.
      * For text length limits, maximum batch size, and supported text encoding, see
      * <a href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/overview#data-limits"/>.
-     * @return the {@link DetectedLanguage detected language} of the text.
+     *
+     * @return The {@link DetectedLanguage detected language} of the text.
+     *
      * @throws NullPointerException if {@code text} is {@code null}.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public DetectedLanguage detectLanguage(String text) {
-        return detectLanguageWithResponse(text, client.getDefaultCountryHint(), Context.NONE).getValue();
+        return detectLanguage(text, client.getDefaultCountryHint());
     }
 
     /**
-     * Returns a {@link Response} containing the detected language and a numeric score between zero and one.
+     * Returns the detected language and a confidence score between zero and one.
      * Scores close to one indicate 100% certainty that the identified language is true.
      *
      * <p><strong>Code Sample</strong></p>
-     * <p>Detects the language with http response in a text with a provided country hint.</p>
-     * {@codesnippet com.azure.ai.textanalytics.TextAnalyticsClient.detectLanguageWithResponse#String-String-Context}
+     * <p>Detects the language of single input text with a provided country hint.</p>
+     * {@codesnippet com.azure.ai.textanalytics.TextAnalyticsClient.detectLanguage#String-String}
      *
      * @param text The text to be analyzed.
      * For text length limits, maximum batch size, and supported text encoding, see
      * <a href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/overview#data-limits"/>.
      * @param countryHint Accepts two letter country codes specified by ISO 3166-1 alpha-2. Defaults to "US" if not
```

Comment:
>Defaults to "US"

Because that is the default from the service? should we instead default it to the default of the client?

Reply:
As an example, in .[NET ](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs#L178)we use the default of the client and then give the option to the user to override the behaviour and use None as default.
Python does similar

Reply:
Java doesn't take default parameter value, I created an issue to address this feedback. #8353